ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -40,10 +40,9 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{include reference="Meeting.Code.Macros"/}}
```

Comment:
This line was added for migrations, in https://github.com/xwikisas/application-meeting/commit/05f2b798beec2f1a3b61bf72904b52ea013d4ec1#diff-2cd615d8cc6bb7c5fc673880e7a0efcbR42. Is not needed anymore.